src/client-side-encryption/stateMachine.ts

```diff
@@ -56,90 +65,149 @@ const INSECURE_TLS_OPTIONS = [
 ];
```

Comment:
```
'tlsDisableOCSPEndpointCheck',
'tlsDisableCertificateRevocationCheck'
```
These are not client options. Maybe we should leave them though but with a comment to help clear up potential confusion.